src/objecttypes/tests/test_objecttype_admin.py

```diff
@@ -129,6 +130,44 @@ def test_create_objecttype_with_invalid_json_schema(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(ObjectType.objects.count(), 0)
 
+    def test_create_objecttype_from_github_url(self):
+        get_response = self.app.get(self.import_github_url)
+
+        form = get_response.form
+        form[
+            "github_url"
+        ] = "https://raw.githubusercontent.com/open-objecten/objecttypes/39d628613b378286cc0b083773df49d9eb55294e/community-concepts/boom/boom-delft.json"
```

Comment:
Actually I don't think that sending real HTTP requests in the unit tests is a good idea. Could you replace it with `requests_mock`call?